config_template.json

```diff
@@ -0,0 +1,57 @@
+{
+    "bitcoind":{
+        "rpcuser": "",
+        "rpcpassword": "",
+        "rpcurl": "",
+        "work_update_seconds": 40,
+        "notify_fallback": true
```

Comment:
This is inefficient and not recommended. Rather not support Docker at all, if we can't find a way around this. (Maybe it can wait for longpoll support?)

Reply:
"notify_fallback" should be false, and bitcoind must then be configured with blocknotify as per README